Formula/cctools.rb

```diff
@@ -113,12 +113,7 @@ def install
     # Fixes build with gcc-4.2: https://trac.macports.org/ticket/43745
     args << "SDK=-std=gnu99"
 
-    if Hardware::CPU.intel?
-      archs = "i386 x86_64"
-    else
-      archs = "ppc i386 x86_64"
-    end
-    args << "RC_ARCHS=#{archs}"
+    args << "RC_ARCHS=i386 x86_64"
```

Comment:
should it have `i386` or just missed?

Reply:
I think it should for now at least as I'm trying to keep things as-is. Could be tweaked for a later PR though 👍